etc/schemas/2022/03/01/ParseDrugMedicationResponse.xsd

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema xmlns="http://www.w3.org/2001/XMLSchema"
+		xmlns:dosageAI20220301="http://www.dkma.dk/dosageAI/xml.schema/2022/03/01"
+		targetNamespace="http://www.dkma.dk/dosageAI/xml.schema/2022/03/01"
+		elementFormDefault="qualified" 
+		attributeFormDefault="unqualified">
+
+	<include schemaLocation="ParseDrugMedication.xsd"/>
+	<element name="ParseDrugMedicationResponse" type="dosageAI20220301:ParseDrugMedicationResponseType">
+		<annotation>
+			<documentation xml:lang="en-GB">The response for parsing a drug medication</documentation>
+			<documentation xml:lang="da-DK">Svar fra forespørgslen om at parse en lægemiddelordination</documentation>
+		</annotation>
+	</element>
+	
+	<complexType name="ParseDrugMedicationResponseType">
+		<sequence>
+			<element name="ParseDrugMedication" type="dosageAI20220301:ParseDrugMedicationType" maxOccurs="unbounded"/>
+		</sequence>
```

Comment:
handle error code and text from AI server, if the input does not result in a DrugMedication result